src/main/java/org/sigmah/server/handler/util/ProjectMapper.java

```diff
@@ -22,369 +22,367 @@
-
-				favoriteUsesSet.add(uDTO);
-			}
-
-			projectDTO.setFavoriteUsers(favoriteUsesSet);
-
-		} else {
-			projectDTO.setFavoriteUsers(null);
-		}
-
-		// ---END----
-
-		sb.append("- CHILDREN: ");
-		sb.append(new Date().getTime() - start);
-		sb.append("ms.\n");
-
-		if (LOG.isDebugEnabled()) {
-			LOG.debug(sb.toString());
-		}
-
-		return projectDTO;
-	}
-
-	public void fillBudget(final Project project, final ProjectDTO projectDTO) throws NumberFormatException {
-		final Query budgetValueQuery = em().createQuery("SELECT v, e FROM Value v JOIN v.element e WHERE v.containerId = :projectId AND e.id IN (:ids)");
-		budgetValueQuery.setParameter("projectId", project.getId());
-		budgetValueQuery.setParameter("ids", em().createQuery("SELECT b.id FROM BudgetElement b").getResultList());
-		
-		final Iterator<?> i = budgetValueQuery.getResultList().iterator();
-		
-		if (i.hasNext()) {
-			Object[] result = (Object[]) i.next();
-			Value budgetValue = (Value) result[0];
-			BudgetElement budgetElement = (BudgetElement) result[1];
-			final Map<Integer, String> values = ValueResultUtils.splitMapElements(budgetValue.getValue());
-			
-			if (budgetElement.getRatioDividend() != null) {
-				if (budgetElement.getRatioDividend().getType() != null) {
-					projectDTO.setRatioDividendType(budgetElement.getRatioDividend().getType());
-				} else {
-					projectDTO.setRatioDividendLabel(budgetElement.getRatioDividend().getLabel());
-				}
-				if (values.get(budgetElement.getRatioDividend().getId().intValue()) != null) {
-					projectDTO.setRatioDividendValue(Double.parseDouble(values.get(budgetElement.getRatioDividend().getId().intValue())));
-				}
-			}
-			
-			if (budgetElement.getRatioDivisor() != null) {
-				if (budgetElement.getRatioDivisor().getType() != null) {
-					projectDTO.setRatioDivisorType(budgetElement.getRatioDivisor().getType());
-				} else {
-					projectDTO.setRatioDivisorLabel(budgetElement.getRatioDivisor().getLabel());
-				}
-				if (values.get(budgetElement.getRatioDivisor().getId().intValue()) != null) {
-					projectDTO.setRatioDivisorValue(Double.parseDouble(values.get(budgetElement.getRatioDivisor().getId().intValue())));
-				}
-			}
-			
-			for (BudgetSubField budgetSubField : budgetElement.getBudgetSubFields()) {
-				if (budgetSubField.getType() != null) {
-					if (values.get(budgetSubField.getId().intValue()) != null) {
-						switch (budgetSubField.getType()) {
-							case PLANNED:
-								projectDTO.setPlannedBudget(Double.parseDouble(values.get(budgetSubField.getId().intValue())));
-								break;
-							case RECEIVED:
-								projectDTO.setReceivedBudget(Double.parseDouble(values.get(budgetSubField.getId().intValue())));
-								break;
-							case SPENT:
-								projectDTO.setSpendBudget(Double.parseDouble(values.get(budgetSubField.getId().intValue())));
-								break;
-							default:
-								break;
-								
-						}
-					}
-					
-				}
-			}
-		}
-	}
-
-}
+
```

Comment:
@numero-six sorry about the following block but I finally understood why this is happening. 

This file was created under Windows and had \r\n line ending. When the license is added by Maven, the first lines are changed with \n ending.

Since Netbeans guess the line ending style from the first lines of the file, it makes it changes the entire file with \n .